soccer/gameplay/plays/testing/TestHeursitc.py

```diff
@@ -0,0 +1,21 @@
+import main
+import robocup
+import behavior
+import constants
+import standard_play
+import tactics.coordinated_pass
+from tactics.find_pass_point import findPassPoint
+from tactics.test_cost_function import testCostFunction
+
+## A test of the pass heuristic system using cost function optimization
+# should pass to around (0.915239, 1.38493)
+class TestHeuristic(standard_play.StandardPlay):
```

Comment:
I don't think this should be a play...

Reply:
This is just a test for the functionality introduced here. It's fine for now.

Reply:
Yup what Kyle said